tests/square_root.py

```diff
@@ -0,0 +1,46 @@
+import cmath  # Importing cmath for complex number operations
+
+class Roots:
```

Comment:
Please create the `Roots` with `x1`, `x2` and `Coefficients` with `a`, `b`, `c`.